project.godot

```diff
@@ -14,7 +14,7 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://src/Actors/Actor.gd"
 }, {
-"base": "Area2D",
+"base": "Position2D",
```

Comment:
Hmm this seems weird - I'm guessing this isn't intentional? Doesn't seem to cause any problems though...

Reply:
Hm, I don't know. Yeah that's not intentional.